pynsot/commands/cmd_protocols.py

```diff
@@ -0,0 +1,472 @@
+# -*- coding: utf-8 -*-
+
+"""
+Sub-command for Protocols.
+
+In all cases ``data = ctx.params`` when calling the appropriate action method
+on ``ctx.obj``. (e.g. ``ctx.obj.add(ctx.params)``)
+
+Also, ``action = ctx.info_name`` *might* reliably contain the name of the
+action function, but still not sure about that. If so, every function could be
+fundamentally simplified to this::
+
+    getattr(ctx.obj, ctx.info_name)(ctx.params)
+"""
+
+from __future__ import unicode_literals
+import logging
+
+from ..vendor import click
+from . import callbacks, types
+from .cmd_networks import DISPLAY_FIELDS as NETWORK_DISPLAY_FIELDS
+
+
+# Logger
+log = logging.getLogger(__name__)
+
+# Ordered list of 2-tuples of (field, display_name) used to translate object
+# field names oto their human-readable form when calling .print_list().
+DISPLAY_FIELDS = (
```

Comment:
Since `DISPLAY_FIELDS` is intended for the default concise view for multiple objects this feels like a lot of fields that will spread too far in a typical terminal and wrap and be kind of ugly to look at.

My gut feeling is to remove `description` and `site` from this list since they are not really all that useful here.

Reply:
The CLI display for these fields should be moved around a little. This is what it looks like on my end currently:

```+---------------------------------------------------------+
| ID   Attributes   Circuit   Device     Interface   Type |
+---------------------------------------------------------+
| 1                 None      foo-bar1   None        bgp  |
| 2                 None      foo-bar1   None        isis |
+---------------------------------------------------------+
```

I would say let's put them in this ordrer: ID, Device, Type, Interface, Circuit, Attributes. 

Example:

```
+---------------------------------------------------------+
| ID   Device     Type   Interface   Circuit   Attributes |
+---------------------------------------------------------+
| 1    foo-bar1   bgp    None        None                 |
| 2    foo-bar1   isis   None        None                 |
+---------------------------------------------------------+
```